### ISSUE TITLE
Problem with downloading songs whose titles have : in them

### ISSUE THREAD

**Issue by ar7x9** [prabaprakash/Saavn-Downloader#24]
URL in question : https://www.jiosaavn.com/album/gunday/rFL,r8r-O7A_

What happens is that the song is shown as downloaded on the terminal, but in actual there is an undeletable file thats created on my hdd with name "Gunday (Rap By" with no extension and 0 KB size, i.e. a corrupted file.

**Comment by inovachrono** [comment]
May be this will be helpful https://stackoverflow.com/questions/1976007/what-characters-are-forbidden-in-windows-and-linux-directory-names

**Comment by ar7x9** [comment]
I know that : is forbidden by windows. I am not a python coder. There is some code that can be created by someone that can validate filenames before creating the file. If I get some time to learn up Python I will try put a snipped so that it can be merged.

**Comment by ar7x9** [comment]
I just went over the code, let me see if I can find a fix. I am gonna add a few lines after line 141 to check and replace those illegal characters.

Edit: 

I managed to fix it.
Add these lines to Download.py after line 141

`filename = filename.replace(":", "-")`
`filename = filename.replace("<", "-")` 
`filename = filename.replace(">", "-")` 
`filename = filename.replace("?", "-")`
`filename = filename.replace("*", "-")` 
`filename = filename.replace("|", "-")`
